tests/integration/nm/conftest.py

```diff
@@ -16,27 +16,3 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
-import pytest
-
-from libnmstate.nm.plugin import NetworkManagerPlugin
-from libnmstate.nm.context import NmContext
-
-
-@pytest.fixture(scope="function")
-def nm_plugin():
-    plugin = NetworkManagerPlugin()
-    yield plugin
-    if plugin.checkpoint:
-        # Ignore failures as the checkpoint might already expired
-        try:
-            plugin.rollback_checkpoint()
-        except Exception:
-            pass
-    plugin.unload()
-
-
-@pytest.fixture(scope="function")
-def nm_context():
-    ctx = NmContext()
-    yield ctx
-    ctx.clean_up()
```

Comment:
Why is this being moved? And if this is needed (I do not see why), I think you should remove the whole file as the remaining is only the header. Thanks!

Reply:
The changes to tests/integration/nm/conftest.py is moving its pytest
fixture to it actual user tests/integration/nm/checkpoint.py

`contest.py` removed.